### ISSUE TITLE
lost GlobalAveragePooling

### ISSUE THREAD

**Issue by Androsimus** [peteryuX/arcface-tf2#37]
In modules/models.py backbones are loaded without pretrained classification head (include_top=False) and then custom OutputLayer is added on the top. Ignoring pretrained classifier means cutting off GlobalAveragePooling layer, but OutputLayer doesn't contain it.

I propose something like this:
`def OutputLayer(embd_shape, w_decay=5e-4, name='OutputLayer', ckpt_path=None):
    """Output Later"""
    def output_layer(x_in):
        x = inputs = Input(x_in.shape[1:])
        # x = BatchNormalization()(x) # maybe this layer is redundunt
        x = GlobalAveragePooling2D()(x)
        x = Dropout(rate=0.1)(x)
        x = Flatten()(x)
        x = Dense(embd_shape, kernel_regularizer=_regularizer(w_decay))(x)
        x = BatchNormalization()(x)
        model = Model(inputs, x, name=name)
        # if ckpt_path:
        #     model.load_weights(ckpt_path)#, by_name=True)
        return model(x_in)
    return output_layer
`
